app/utils/template-helpers.ts

```diff
@@ -0,0 +1,39 @@
+import config from "../config";
+
+const templateHelpers = {
+  dateFormat: (date: string): string =>
+    new Date(date).toLocaleDateString("en-CA", {
+      year: "numeric",
+      month: "numeric",
+      day: "numeric",
+    }),
+  eq: (a: string, b: string): boolean => a == b,
+  updateMarkup: (content: string | undefined): string => {
+    if (!content) return "";
+
+    const siteUrl = new RegExp(`${config.wordpress.url}`, "g");
+
+    // Updates the WordPress content to match with the expected markup
```

Comment:
Is there something we can embed upstream? i.e., do you foresee this logic being used by other internal dynamic websites?

Reply:
Ideally we'd have Articles outputting valid Design System markdown since it would be cleaner/simpler.  

@willeybryan captured an issue here for that work:
https://github.com/cds-snc/platform-core-services/issues/673